imagine/fields/field.py

```diff
@@ -68,6 +68,12 @@ def type(self):
         """Type of the field"""
         return(self.TYPE)
 
+    @property
+    @req_attr
+    def parameters_list(self):
+        """Parameters of the field"""
+        return self.PARAMETERS_LIST
+
```

Comment:
I would name this 'parameter_names' to avoid confusion with the `parameters` property.

Reply:
The original plan was calling it `required_parameters_list`.  While doing it, I felt it was too verbose....

But you are right, `parameter_names` makes a lot more sense!